src/DssSpell.sol

```diff
@@ -18,52 +18,262 @@ pragma solidity 0.6.12;
 
-    // Turn off office hours
-    function officeHours() public override returns (bool) {
-        return false;
-    }
+    // Many of the settings that change weekly rely on the rate accumulator
+    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
+    // To check this yourself, use the following rate calculation (example 8%):
+    //
+    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
+    //
+    // A table of rates can be found at
+    //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
+    //
+    uint256 constant TWO_PCT            = 1000000000627937192491029810;
+    uint256 constant FOUR_PT_FIVE_PCT   = 1000000001395766281313196627;
+    uint256 constant SIX_PCT            = 1000000001847694957439350562;
+    uint256 constant SEVEN_PCT          = 1000000002145441671308778766;
 
-    uint256 constant WAD     = 10 ** 18;
-    uint256 constant MILLION = 10 ** 6;
+    // Math
+    uint256 constant THOUSAND = 10 ** 3;
+    uint256 constant MILLION  = 10 ** 6;
+    uint256 constant WAD      = 10 ** 18;
+    uint256 constant RAY      = 10 ** 27;
+    uint256 constant RAD      = 10 ** 45;
 
-    address constant MIP21_LIQUIDATION_ORACLE = 0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30;
+    // Maker changelog 
+    address public constant MAKER_CHANGELOG = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
 
     function actions() public override {
+        // https://vote.makerdao.com/polling/QmRCn7Mh#poll-detail
+        CentrifugeCollateralValues memory RWA003 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x1Fe789BBac5b141bdD795A3Bc5E12Af29dDB4b86,
+            GEM: 0x07F0A80aD7AeB7BfB7f139EA71B3C8f7E17156B9,
+            OPERATOR: 0x2A9798c6F165B6D60Cfb923Fe5BFD6f338695D9B,
+            INPUT_CONDUIT: 0x2A9798c6F165B6D60Cfb923Fe5BFD6f338695D9B,
+            OUTPUT_CONDUIT: 0x2A9798c6F165B6D60Cfb923Fe5BFD6f338695D9B,
+            URN:  0x7bF825718e7C388c3be16CFe9982539A7455540F,
+            gemID: "RWA003",
+            joinID: "MCD_JOIN_RWA003_A",
+            urnID: "RWA003_A_URN",
+            inputConduitID: "RWA003_A_INPUT_CONDUIT",
+            outputConduitID: "RWA003_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA003",
+            ilk: "RWA003-A",
+            ilk_string: "RWA003-A",
+            ilkRegistryName: "RWA003-A: Centrifuge: ConsolFreight",
+            RATE: SIX_PCT,
+            CEIL: 2 * MILLION,
+            PRICE: 2_359_560 * WAD,
+            MAT: 10_500,
+            TAU: 0,
+            DOC: "QmQMNfSbGS8qkJbatQgxMUsz27G8YELWgtXeLs8uFCZoY8"
+        });
+
+        // https://vote.makerdao.com/polling/QmRWg6nZ#poll-detail
+        CentrifugeCollateralValues memory RWA004 = CentrifugeCollateralValues({
+            MCD_JOIN: 0xD50a8e9369140539D1c2D113c4dC1e659c6242eB,
+            GEM: 0x873F2101047A62F84456E3B2B13df2287925D3F9,
+            OPERATOR: 0xe1ed3F588A98bF8a3744f4BF74Fd8540e81AdE3f,
```

Comment:
ditto

Reply:
Likewise, we'll do that in the integration spell we'll run on our side after the executive vote.